main/ServiceManager/ServiceManager.cpp

```diff
@@ -129,11 +129,17 @@ void ServiceMangerTaskInit()
  */
 void ServiceMangerInit()
 {
+    nvsFlashInit();
+
     if (SharedBusInit())
```

Comment:
another wrong `if` statement here. This is how it should be written:
```
uint8_t init = SharedBusInit();
if (!init)
{
    ESP_LOGE(TAG, "Failed to Initialize SharedBus.")
}

ESP_LOGI(TAG, "initialized SharedBus successfully");

Reply:
1- your suggestion is wrong in the "else" state    
2- I am not happy with this style
3- what you ask for is totally the same as what I have done   
ANYWAY fixed and done